services/app-api/forms/mcpar.json

```diff
@@ -3407,6 +3407,138 @@
               }
             ]
           }
+        },
+        {
+          "name": "XI: ILOS",
+          "path": "/mcpar/plan-level-indicators/ilos",
+          "pageType": "drawer",
+          "entityType": "plans",
+          "verbiage": {
+            "intro": {
+              "section": "Section D: Plan-Level Indicators",
+              "subsection": "Topic XI. ILOS",
+              "spreadsheet": "D4_Plan_ILOS",
+              "info": [
+                {
+                  "type": "span",
+                  "content": "If In Lieu of Services and Settings (ILOS) are available in this managed care program, report ILOS utilization for each plan during the contract rating period. Select \"Enter\" for each plan listed below to indicate whether or not ILOS were offered by the plan, select which ILOS the plan offered, and enter data on utilization for each ILOS offered."
+                }
+              ]
+            },
+            "dashboardTitle": "Report ILOS utilization for each managed care plan during the contract rating period.",
+            "drawerTitle": "",
+            "missingEntityMessage": [
+              {
+                "type": "span",
+                "content": "This program is missing plans. You won’t be able to complete this section until you’ve added all the plans that serve enrollees in the program. "
```

Comment:
Nit: This has the smart `’` in `won’t` and `you’ve` while a straight apostrophe `'` is used in other text.